app/src/main/java/com/vero/tasky/TaskyApplication.kt

```diff
@@ -15,4 +19,22 @@ class TaskyApplication : Application(), Configuration.Provider {
     override fun getWorkManagerConfiguration(): Configuration {
         return Configuration.Builder().setWorkerFactory(workerFactory).build()
     }
+
+    override fun onCreate() {
+        super.onCreate()
+        createNotificationChannel()
```

Comment:
Are you targetting apis lower than 26?
If so, this code needs to run on api26(oreo) or higher

Or else, it will crash on apis lower than that

Reply:
Currently, I'm targeting API higher than 26,